f3dex2.s

```diff
@@ -1930,27 +2048,27 @@ G_MTX_handler:
     andi    $11, cmd_w0, G_MTX_P_MV | G_MTX_NOPUSH_PUSH // Read the matrix type and push type flags into $11
     bnez    $11, load_mtx                               // If the matrix type is projection or this is not a push, skip pushing the matrix
      andi   $2, cmd_w0, G_MTX_MUL_LOAD                  // Read the matrix load type into $2 (0 is multiply, 2 is load)
-    lw      matrixStackAddr, matrixStackPtr             // Set up the DMA from dmem to rdram at the matrix stack pointer
-    li      $20, -0x2000                                //
+    lw      cmd_w1_dram, matrixStackPtr                 // Set up the DMA from dmem to rdram at the matrix stack pointer
+    li      dmemAddr, -0x2000                           //
     jal     dma_read_write                              // DMA the current matrix from dmem to rdram
-     li     $19, 0x0040 - 1                             // Set the DMA length to the size of a matrix (minus 1 because DMA is inclusive)
-    addi    matrixStackAddr, matrixStackAddr, 0x40      // Increase the matrix stack pointer by the size of one matrix
-    sw      matrixStackAddr, matrixStackPtr             // Update the matrix stack pointer
-    lw      cmd_w1, (inputBufferEnd - 4)(inputBufferPos)
+     li     dmaLen, 0x0040 - 1                          // Set the DMA length to the size of a matrix (minus 1 because DMA is inclusive)
+    addi    cmd_w1_dram, cmd_w1_dram, 0x40              // Increase the matrix stack pointer by the size of one matrix
```

Comment:
Same as above.

Reply:
Gonna have to disagree on this one--in this case, the register is explicitly being used as the DRAM address during this code, including for `dma_read_write`. So this really is the right name for it.

Reply:
That's fair, it's a pretty good reason to use the new name